docker/entry.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+sa_dir=/var/run/secrets/kubernetes.io/serviceaccount
+
+kc_args="--server=https://kubernetes.default.svc.cluster.local --token=$(cat $sa_dir/token) --certificate-authority=$sa_dir/ca.crt"
+
+kubectl $kc_args create -f manifests/
```

Comment:
You'll need to test upgrade...
- using `create`, the command will fail if the objects already exist
- if you use `apply`, it will permit patching the already existing resources. The thing to check is whether the resource types you're installing can actually be patched.
- as a fallback, you can `delete` then `create`, but that'll cause problems during the changeover.

Reply:
@JohnStrunk I will test the patching scenario and also will change it to apply if that works. 

> as a fallback, you can delete then create, but that'll cause problems during the changeover.

I agree, delete and creating will be a fallback scenario. I think the patching should happen with kubectl apply will test and update.